crates/bevy_render/src/render_graph/nodes/render_resources_node.rs

```diff
@@ -420,6 +421,7 @@ fn render_resources_node_system<T: RenderResources>(
     mut state: Local<RenderResourcesNodeState<T>>,
     render_resource_context: Res<Box<dyn RenderResourceContext>>,
     mut query: Query<(&T, &Draw, &mut RenderPipelines)>,
+    mut query2: Query<(&T, &Dispatch, &mut ComputePipelines)>,
```

Comment:
nit: as a style thing, whenever we have more than one query, give them names to describe them. ex: `draw_query` and `compute_query`